doc/datasets/openml.rst

```diff
@@ -0,0 +1,155 @@
+
+  >>> print(mice.url)
+  https://www.openml.org/d/40966
+
+The id is also the most specific way to specify how to fetch a dataset from
+OpenML::
+
+  >>> mice = fetch_openml(id=40966, data_home=custom_data_home)
+  >>> mice.details # doctest: +NORMALIZE_WHITESPACE +ELLIPSIS +SKIP
+  {'id': '4550', 'name': 'MiceProtein', 'version': '1', 'format': 'ARFF',
+  'creator': ...,
+  'upload_date': '2016-02-17T14:32:49', 'licence': 'Public', 'url':
+  'https://www.openml.org/data/v1/download/1804243/MiceProtein.ARFF', 'file_id':
+  '1804243', 'default_target_attribute': 'class', 'citation': 'Higuera C,
+  Gardiner KJ, Cios KJ (2015) Self-Organizing Feature Maps Identify Proteins
+  Critical to Learning in a Mouse Model of Down Syndrome. PLoS ONE 10(6):
+  e0129126. [Web Link] journal.pone.0129126', 'tag': ['OpenML100', 'study_14',
+  'study_34'], 'visibility': 'public', 'status': 'active', 'md5_checksum':
+  '3c479a6885bfa0438971388283a1ce32'}
+
+.. _openml_versions:
+
+Dataset Versions
+----------------
+
+A dataset is uniquely specified by its id, but not necessarily by its name.
+Several different "versions" of a dataset with the same name can exist which can contain
+entirely different datasets.
+If a particular version of a dataset has been found to contain significant
+issues, it might be inactivated. Using a name to specify a dataset will yield
+the earliest version of a dataset that is still active. That means that
+``fetch_openml(name="miceprotein")`` can yield different results at different
+times if earlier versions become inactive.
+You can see that the dataset with id 40966 that we fetched above is the version 1
+of the "miceprotein" dataset::
+
+  >>> mice.details['version']  #doctest: +SKIP
+  '1'
+
+In fact, this dataset only has one version. The iris dataset on the other hand
+has multiple versions::
+
+  >>> iris = fetch_openml(name="iris", data_home=custom_data_home)
+  >>> iris.details['version']  #doctest: +SKIP
+  '1'
+  >>> iris.details['id']  #doctest: +SKIP
+  '61'
+
+  >>> iris_61 = fetch_openml(id=61, data_home=custom_data_home)
+  >>> iris_61.details['version']  #doctest: +SKIP
+  '1'
+  >>> iris_61.details['id']  #doctest: +SKIP
+  '61'
+
+  >>> iris_969 = fetch_openml(id=969, data_home=custom_data_home)
+  >>> iris_969.details['version']  #doctest: +SKIP
+  '3'
+  >>> iris_969.details['id']  #doctest: +SKIP
+  '969'
+
+Specifying the dataset by the name "iris" yields the lowest version, version 1, with the id 61.
+To make sure you always get this exact dataset, it is safest to specify it by the dataset id.
+The other dataset, with id 969, is version 3 (version 2 has become inactive), and contains
+a binarized version of the data::
+
+  >>> np.unique(iris_969.target)  #doctest: +SKIP
+  array([b'N', b'P'],
+        dtype='|S1')
+
+You can also specify both the name and the version, which also uniquely identifies the dataset:: 
+  >>> iris_version_3 = fetch_openml(name="iris", version=3, data_home=custom_data_home)
```

Comment:
need an empty line above I think?

Reply:
I don't quite understand what you mean by empty link?